src/include/sof/lib/watchdog.h

```diff
@@ -0,0 +1,27 @@
+/* SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Care to add some explanation in the commit message on *why* this is needed and how it will be used?

Reply:
Added description in header file.